config/Config.h

```diff
@@ -22,6 +22,7 @@ class Config : public ConfigReader
     CFG_PROP(uint32_t, defaultLastN, 5);
     CFG_PROP(uint32_t, dropInboundAfterInactive, 3);
     CFG_PROP(uint32_t, maxDefaultLevelBandwidthKbps, 3000);
+    CFG_PROP(uint32_t, rtpForwardInterval, 10); // ms
```

Comment:
I noticed we had hardcoded 2ms before.  Do you think isn't too much? I mean we still have video use cases where we can receive a lot of packet for same fame. It can happens we process some packets and the others that arrive on millisecond later have to wait 10 ms to be forwarder.